screep

```diff
@@ -272,7 +272,7 @@ if args.subcommand == 'echo':
                   " does not exist. Exiting.")
             sys.exit()
 
```

Comment:
Oh noes! Echo got left out of the fun here when we moved to passing single variable names to encode and decode instead of lists. Thus issue #44